crates/loam-sdk/README.md

```diff
@@ -0,0 +1,121 @@
+# loam-sdk
+
+- [Getting Started](#getting-started)
+    - [Installation](#installation)
+    - [Setup](#setup)
+    - [Redeploy](#redeploy)
+- [Subcontracts](#subcontracts)
+    - [Creating Contract Subcontracts](#creating-contract-subcontracts)
+    - [External API](#external-api)
+- [CoreSubcontract](#coreriff)
+-   [Using the Core Subcontract](#using-the-coreriff)
+
+### Installation
+
+To install `just`, run the following command:
+
+```bash
+cargo install just
+```
+
+### Setup
+
+To set up the environment, run:
+
+```bash
+just setup
+```
+
+### Redeploy
+
+To see redeployment in action, use:
+
+```bash
+just redeploy
+```
```

Comment:
@willemneal this seems like stuff that would be better in a `CONTRIBUTING.md` file. That is, it seems like info about how to contribute to `loam-sdk`, not how to install it. Is that correct?